lib/session_redis_down_error_handler.rb

```diff
@@ -0,0 +1,5 @@
+class SessionRedisDownErrorHandler
+  def self.call(error, _env, _sid)
+    raise error
+  end
+end
```

Comment:
now that we inlined with a proc, we can remove this file, right?

Reply:
yep good catch, thanks, removed